### ISSUE TITLE
Alternative (dxgi) installation problem

### ISSUE THREAD

**Issue by donizettilorenzo** [cdozdil/OptiScaler#78]
It just says this:

https://i.postimg.cc/T1nMBZ4V/Untitled.jpg

**Comment by cdozdil** [comment]
Enter the game and check again

**Comment by donizettilorenzo** [comment]
Same output....

**Comment by cdozdil** [comment]
Maybe you 2 copies of Opti in game folder?

**Comment by donizettilorenzo** [comment]
Nope, maybe it's the dlss DLL? No problem with Unreal Engine 5 games (where the DLSS DLSS is not in the main exe folder)

**Comment by vyvertyt** [comment]
Same in Cyberpunk

**Comment by cdozdil** [comment]
Hi,

Could you send a screenshot of intallation folder. Cyberpunk is one of the most tested games with this mod and it's working ok.

BTW, I would like to remind a general tip. If game supports DLSS-FG using [DLSS Enabler](https://github.com/artur-graniszewski/DLSS-Enabler) would almost always gives better results.

**Comment by TheRazerMD** [comment]
Btw checked with Optiscaler 0.7.7-pre4, and HZD works fine.
